### PR TITLE
Add parser plugin helper document

### PR DESCRIPTION
**Pull request by okkez**

### CODE REVIEW

File: docs/v0.14/api-plugin-helper-parser.txt [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+        # 1. load parser helper
+        helpers :parser
+
+        # omit shutdown and other plugin API
+        
+        def configure(conf)
+          super
+          
+          @parser = parser_create
```
Comment (repeatedly) [inline docs/v0.14/api-plugin-helper-parser.txt]: Adding `usage` is better for example?

Comment (okkez) [reply docs/v0.14/api-plugin-helper-parser.txt]: Yes! Added usage.

File: docs/v0.14/api-plugin-helper-parser.txt [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    :::ruby
+    # Create parser plugin instance using <parse> section in fluent.conf during configure phase
+    @parser = parser_create
+    @parser.parse(text) do |time, record|
+      ...
+    end
+
+    # Create JSON parser
+    @json_parser = parser_create(usage: 'parser_in_example_json', type: 'json')
+    @json_parser.parse do |time, record|
```
Comment (repeatedly) [inline docs/v0.14/api-plugin-helper-parser.txt]: Missing parameter for `parse`.

**Comment by repeatedly** [comment]
Adding link to parser plugin article is better.

File: docs/v0.14/api-plugin-helper-parser.txt (hunk 2 above)
Comment (okkez) [reply docs/v0.14/api-plugin-helper-parser.txt]: Oh! Thanks!

**Comment by okkez** [comment]
Added link to parser plugin articles.

**Comment by repeatedly** [comment]
Thx!

**Pull request merged**